wafer/talks/models.py

```diff
@@ -268,3 +274,51 @@ class TalkUrl(models.Model):
     description = models.CharField(max_length=256)
     url = models.URLField()
     talk = models.ForeignKey(Talk, on_delete=models.CASCADE)
+
+
+@python_2_unicode_compatible
+class Review(models.Model):
+    talk = models.ForeignKey(Talk, on_delete=models.CASCADE,
+                             related_name='reviews')
+    reviewer = models.ForeignKey(settings.AUTH_USER_MODEL,
+                                 on_delete=models.CASCADE)
+
+    notes = models.TextField(
+        null=True, blank=True,
+        help_text=_("Comments on the proposal"))
```

Comment:
Would this perhaps be better as a MarkupField, so we can have formatted reviews and all that?

Reply:
Possibly - I'm more ambivalent about changing notes, but I can see an argument for it.